sdk/storage/azure_storage_blob/src/generated/clients/append_blob_client.rs

```diff
@@ -15,9 +15,9 @@ use azure_core::{
     error::CheckSuccessOptions,
     fmt::SafeDebug,
     http::{
-        policies::{auth::BearerTokenAuthorizationPolicy, Policy},
+        policies::{BearerTokenAuthorizationPolicy, Policy},
```

Comment:
>unresolved import `azure_core::http::policies::BearerTokenAuthorizationPolicy`
consider importing this struct instead:
azure_core::http::policies::auth::BearerTokenAuthorizationPolicy

Not sure why it is now pointing to a different path for the fully qualified name?